source/Calamari.Testing/EnvironmentVariables.cs

```diff
@@ -11,73 +11,75 @@ namespace Calamari.Testing
         DockerReaderPassword,
 
-        [EnvironmentVariable("AWS_E2E_AccessKeyId", "AWS E2E Test User Keys")]
+        [EnvironmentVariable("AWS_E2E_AccessKeyId", "op://Calamari Secrets for Tests/AWS E2E Test User Keys/AccessKeyId")]
         AwsCloudFormationAndS3AccessKey,
 
-        [EnvironmentVariable("AWS_E2E_SecretKeyId", "AWS E2E Test User Keys")]
+        [EnvironmentVariable("AWS_E2E_SecretKeyId", "op://Calamari Secrets for Tests/AWS E2E Test User Keys/SecretKeyId")]
         AwsCloudFormationAndS3SecretKey,
 
-        [EnvironmentVariable("CALAMARI_FEEDZV2URI", "Not LastPass; Calamari TC Config Variables")]
+        [EnvironmentVariable("CALAMARI_FEEDZV2URI", defaultValue: "https://f.feedz.io/octopus-deploy/integration-tests/nuget")]
         FeedzNuGetV2FeedUrl,
 
-        [EnvironmentVariable("CALAMARI_FEEDZV3URI", "Not LastPass; Calamari TC Config Variables")]
+        [EnvironmentVariable("CALAMARI_FEEDZV3URI", defaultValue: "https://f.feedz.io/octopus-deploy/integration-tests/nuget/index.json")]
         FeedzNuGetV3FeedUrl,
 
-        [EnvironmentVariable("CALAMARI_ARTIFACTORYV2URI", "Not LastPass; Calamari TC Config Variables")]
+        [EnvironmentVariable("CALAMARI_ARTIFACTORYV2URI", defaultValue: "https://nuget.packages.octopushq.com/")]
         ArtifactoryNuGetV2FeedUrl,
 
-        [EnvironmentVariable("CALAMARI_ARTIFACTORYV3URI", "Not LastPass; Calamari TC Config Variables")]
+        [EnvironmentVariable("CALAMARI_ARTIFACTORYV3URI", defaultValue: "https://packages.octopushq.com/artifactory/api/nuget/v3/nuget")]
         ArtifactoryNuGetV3FeedUrl,
 
-        [EnvironmentVariable("CALAMARI_AUTHURI", "OctopusMyGetTester")]
+        [EnvironmentVariable("CALAMARI_AUTHURI", "op://Calamari Secrets For Tests/MyGet Package Manager/website")]
         MyGetFeedUrl,
 
-        [EnvironmentVariable("CALAMARI_AUTHUSERNAME", "OctopusMyGetTester")]
+        [EnvironmentVariable("CALAMARI_AUTHUSERNAME", "op://Calamari Secrets For Tests/MyGet Package Manager/username")]
         MyGetFeedUsername,
 
-        [EnvironmentVariable("CALAMARI_AUTHPASSWORD", "OctopusMyGetTester")]
+        [EnvironmentVariable("CALAMARI_AUTHPASSWORD", "op://Calamari Secrets For Tests/MyGet Package Manager/password")]
         MyGetFeedPassword,
 
-        [EnvironmentVariable("GOOGLECLOUD_OCTOPUSAPITESTER_JSONKEY", "GoogleCloud - OctopusAPITester", "op://Calamari Secrets for Tests/Google Cloud Octopus Api Tester JsonKey")]
+        [EnvironmentVariable("GOOGLECLOUD_OCTOPUSAPITESTER_JSONKEY", "op://Calamari Secrets for Tests/Google Cloud Octopus Api Tester/jsonkey")]
         GoogleCloudJsonKeyfile,
 
-        [EnvironmentVariable("GitHub_RateLimitingPersonalAccessToken", "GitHub test account PAT")]
+        //TODO(tmm): not sure about this one - this is a copy of the github account above.
+        [EnvironmentVariable("GitHub_RateLimitingPersonalAccessToken", "op://Calamari Secrets for Tests/GitHub Test Account/PAT")]
```

Comment:
Is there anyone we can talk to about this? Looks like @IsaacCalligeros95 set this up originally?

Reply:
This variable is used in the test setup InstallTools to get the aws-iam-authenticator. It was available in LastPass but doesn't look to have made it to 1Password. I just used a separate token to avoid any potential rate limiting on the existing tokens but we can probably just use any of the existing GH PAT variables in teamcity.  It can be any user token and doesn't require any permissions, it's just for that install step.

Reply:
Thanks for the context @IsaacCalligeros95!